src/api/v4/client.js

```diff
@@ -15,8 +15,12 @@ function getValidationError (code) {
 /**
  * This is the entry point for a CARTO.js application.
  *
- * A CARTO client allows managing layers and dataviews. It also takes care
- * of the communication between a CARTO.js application and the services in CARTO.
+ * A CARTO client allows managing layers and dataviews. Once you add a layer or a dataview, the client 
+ * handles the communication against the server in order to process those data.
```

Comment:
"...the client handles the communication against the server in order to process those data"... 🤔